src/status_im2/subs/wallet/wallet.cljs

```diff
@@ -71,21 +71,18 @@
  :<- [:wallet/current-viewing-account]
  :<- [:wallet/network-details]
  (fn [[account networks] [_ query]]
-   (let [tokens (map (fn [token]
-                       (assoc token
-                              :networks           (utils/network-list token networks)
-                              :total-balance      (utils/total-token-value-in-all-chains token)
-                              :total-balance-fiat (utils/calculate-balance token)))
-                     (:tokens account))
-
-         sorted-tokens
-         (sort-by :name compare tokens)
-         filtered-tokens
-         (filter #(or (string/starts-with? (string/lower-case (:name %))
-                                           (string/lower-case query))
-                      (string/starts-with? (string/lower-case (:symbol %))
-                                           (string/lower-case query)))
-                 sorted-tokens)]
+   (let [tokens          (map (fn [token]
+                                (assoc token
+                                       :networks           (utils/network-list token networks)
+                                       :total-balance      (utils/total-token-units-in-all-chains token)
+                                       :total-balance-fiat 0))
```

Comment:
@churik 
For the wallet flow:
1. Select an account
2. Select Send button
3. Write an address (0x123..)
4. Select an address

the fiat balance is being returned as zero as part of this PR.

The reason is that the value has a bug in develop, and its calculation was  perfomed using code that I erased in this PR, so instead of fixing that bug here, I created an issue:
- #18067

to properly address it. And set it as zero for this PR.
cc @briansztamfater